force_wfmanager/run.py

```diff
@@ -19,7 +19,8 @@
 push_exception_handler(lambda *args: None, reraise_exceptions=True)
 
 
-@click.command()
+@click.command(context_settings=dict(ignore_unknown_options=True,
+                                     allow_extra_args=True))
```

Comment:
Why do you need them?

Reply:
I was just trying it to see if it made the integration test work. The version with argparse passed when I let it ignore unexpected arguments so I thought that was the issue here. But clearly it's not the case, perhaps because click passes an exit code which the test doesn't expect?

Reply:
You can't invoke a click decorated function as a function anymore. Either extract the code in a separate _main() that acts like a function and test that one, or use the click CliRunner, but the first solution is probably what you need